### PR TITLE
[FIX] point_of_sale: disable double-tap zooming

### PR DESCRIPTION
**Pull request by caburj**
Check: https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action#manipulation

### CODE REVIEW

File: addons/point_of_sale/static/src/app/pos_app.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
        -moz-user-select: none;
             user-select: none;
     text-shadow: none;
+    touch-action: manipulation;
```
Comment (vlst-odoo) [inline addons/point_of_sale/static/src/app/pos_app.scss:13]: wouldn't it be better to disable zooming completely ? Even pinch to zoom

Comment (caburj) [reply addons/point_of_sale/static/src/app/pos_app.scss:13]: > wouldn't it be better to disable zooming completely ? Even pinch to zoom

I think that's a bad UX. Some people (including me) tend to pinch zoom.

Comment (mobt-odoo) [reply addons/point_of_sale/static/src/app/pos_app.scss:13]: @vlst-odoo @caburj Do not restrict the pinch zoom

**Review by vlst-odoo — APPROVED** [review]
robodoo r+

**Pull request closed without merging**